### PR TITLE
Transcript config

### PR DESCRIPTION
**Pull request by robhaswell**

### CODE REVIEW

File: TENTATIVE-TRANSCRIPTS-v0.2.rst [hunk 1]
```diff
@@ -120,6 +120,59 @@ transcript::
...
+configuration
+-------------
+
+get a key
+~~~~~~~~~
+
+transcript::
+
+    $ dvol config user.name
+    Rob Haswell
```
Comment (lukemarsden) [inline TENTATIVE-TRANSCRIPTS-v0.2.rst]: `>` prompt is normal

File: TENTATIVE-TRANSCRIPTS-v0.2.rst [hunk 2]
```diff
@@ -120,6 +120,59 @@ transcript::
...
+identification
+--------------
+
+unidentified commit operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+transcript::
+
+    $ dvol commit -m'A message'
+    You have not identified yourself. Log into a volume hub or using dvol config:
```
Comment (lukemarsden) [inline TENTATIVE-TRANSCRIPTS-v0.2.rst]: `Log into volume hub with dvol login`

File: TENTATIVE-TRANSCRIPTS-v0.2.rst [hunk 3]
```diff
@@ -120,6 +120,59 @@ transcript::
...
+
+        dvol config user.name "My Name"
+        dvol config user.email
```
Comment (lukemarsden) [inline TENTATIVE-TRANSCRIPTS-v0.2.rst]: missing example email address

**Comment by lukemarsden** [comment]
Generally looks good. I think `dvol config --help` which shows which keys can be set would aid discoverability. Otherwise, please address my comments and then merge. Thanks!

**Comment by robhaswell** [comment]
@lukemarsden can I get a re-review with the new text please.

Re `--help` I agree, but I don't know what is yet possible with Cobra. I will bear this in mind when I am developing this.

**Comment by lukemarsden** [comment]
Oh, yeah, I was happy for you to merge after addressing my comments to your satisfaction. Thanks, merging now.

**Pull request merged**